src/runtime/LegacyMeta.svelte

```diff
@@ -0,0 +1,14 @@
+<script lang="ts">
```

Comment:
All these legacy components could just throw an error in their instance script right?
If they actually exist at runtime, it must be because the user hans't enabled the legacy flag right?

Perhaps a nice documented error.

Reply:
Very good point, you're right.

I think adding a checker inside the `index.ts` while importing the legacy components is hard _(no idea if there's a simple way to access addon options)_.
So I can do it while traversing on the Svelte AST inside the indexer parser and post-transform maybe _(because pre-transform will be skipped)_.

Reply:
> no idea if there's a simple way to access addon options

It's doable, but not simple so let's try to avoid it.

> I can do it while traversing on the Svelte AST inside the indexer parser and post-transform maybe (because pre-transform will be skipped without having this option enabled).

Just making the components throw an error should be the simplest solution here right? Like

```svelte
<script>
  throw new Error("you have not enabled the legacy option...");
</script>
```

Reply:
That's much simpler, thanks for the hint!

Do we need to have test for it? No idea how we could do it in this repo _(with one Storybook config)_ and not breaking the CI(?)

Reply:
Not easily no, I'm fine with this just having been tested manually.

Reply:
In commit: 34d0eacc633683e850a663b1e65b7734f01658cd

I added this error, but after testing manually I don't think this was the right direction 😭 

The first error I got is:
```
storybook v8.2.5

info => Cleaning outputDir: storybook-static
info => Loading presets
info => Building manager..
info => Manager built (116 ms)
info => Building preview..
WARN 🚨 Unable to index ./tests/stories/LegacyWithInstance.stories.svelte:
WARN   SB_SVELTE_CSF_PARSER_EXTRACT_SVELTE_0001 (MissingModuleTagError): The file 'file:///Users/xeho91/Nextcloud/Projects/oss/addon-svelte-csf/tests/stories/LegacyWithInstance.stories.svelte'
WARN does not have a module context (<script context="module"> ... </script>).
WARN
WARN defineMeta(...) should be called inside a module script tag, like so:
WARN
WARN <script context="module">
WARN import { defineMeta } from "@storybook/addon-svelte-csf";
WARN
WARN const { Story } = defineMeta({});
WARN </script>
WARN
WARN More info: https://github.com/storybookjs/addon-svelte-csf/blob/v4.1.2/ERRORS.md#SB_SVELTE_CSF_PARSER_EXTRACT_SVELTE_0001
```

Which makes sense, because indexer parser is being run first. So I guess we have to move error to this parser, which would benefit us by giving us `filename` path.

Reply:
Your idea of using try catch and recognise known errors from indexer parser related to the legacy API _(https://github.com/storybookjs/addon-svelte-csf/commit/95a4becc4191af91533823a45a7047616d0c4b29)_ does the job - tested manually, thanks! 🙏